application/constants/ethereum.go

```diff
@@ -6,12 +6,13 @@
-	TokenWrapperAddress         = "0xA9739b5BdAfe956DEAa8b2e695c7d4f1DF7Bc1D6"
 	TokenWrapperWithdrawUTokens = "withdrawUTokens"
+
+	EthereumBlockConfirmations = 12
+
+	EthereumEmptyTxHash = "0x0000000000000000000000000000000000000000000000000000000000000000"
```

Comment:
It looks hardcoded. Could we use values from `ethereum.common` package?

Reply:
Yes, will fix that